### PR TITLE
Kelechi header

### PR DESCRIPTION
**Pull request by itskelechi**

### CODE REVIEW

File: public/images/tap-logo-light.png
Comment (cengique) [inline public/images/tap-logo-light.png:1]: @itskelechi  I had cropped the logo file to reduce the whitespace around it. Did you mean to leave it square?

File: src/components/ProjectCard.vue [hunk 1]
```diff
@@ -153,13 +153,13 @@ const date_options = {
...
-  /*pointer-events: auto;*/ 
-}
+/* .projectCard {
+  pointer-events: auto; 
+} */
 
 /* Date shown to the right and small */
 .dateStamp {
-  text-align: right;
+  text-align: left;
```
Comment (cengique) [inline src/components/ProjectCard.vue:162]: I had pushed the date to the right so we use the blank space on the top right corner of the project card. But it's up to you.

File: src/components/vue/Breadcrumb.vue [hunk 2]
```diff
@@ -47,6 +47,7 @@ const crumbClasses = (index) => {
 </script>
 <style module>
 .breadcrumb {
+  content: '>';
```
Comment (cengique) [inline src/components/vue/Breadcrumb.vue:50]: why did you remove the > from the breadcrumb? looks weird now

File: src/content/projects/2019-08Fall/BallIsLife/BallIsLife.mdx [hunk 3]
```diff
@@ -79,23 +83,23 @@ The Technology Ambassador Program at [GGC](http://www.ggc.edu/tap) strives to br
 
 {/* content block, Activities, steps & materials */}
 
-<div style="text-align:center; margin-top:50px">
+{/* <div class="prj-txt"> */}
```
Comment (cengique) [inline src/content/projects/2019-08Fall/BallIsLife/BallIsLife.mdx:86]: excellent, I like that we no longer have to provide CSS for each paragraph in the project.

File: src/content/projects/2023-01Spring/Foe/foe-logo-dark.png
Comment (cengique) [inline src/content/projects/2023-01Spring/Foe/foe-logo-dark.png:1]: OMG thank you so much for fixing these logos!

File: src/pages/index.astro [hunk 4]
```diff
@@ -72,13 +72,89 @@ const blogEntries = await getCollection('events');
...
             <PapersListCard client:load/-->
         </section>
+
+        	<!-- <section class="banner">
+		<h2> About TAP </h2>
+	</section> -->
+  
+</TAPDefaultLayout>
+
+<script>
```
Comment (cengique) [inline src/pages/index.astro:101]: we should make this into a component later.

File: src/styles/global.css [hunk 5]
```diff
@@ -28,24 +70,37 @@ body {
...
+  margin: 1rem;
+  padding: 3rem;
+}
 
 /* Make all markdown paragraphs in the blog and projects sections in a centered box */
 .blog p, .body-prj p, .body-prj ol {
-  max-width: 60em;
-  margin-left: auto;
-  margin-right: auto;
+  max-width: 1440px; /*keep width with content*/
```
Comment (cengique) [inline src/styles/global.css:86]: you should not need this since the container already has a max

File: src/styles/global.css [hunk 6]
```diff
@@ -130,16 +265,74 @@ justify-content: space-around;
   font-size: clamp(var(--agnostic-h5), 2vw, var(--agnostic-h3));
 }
 
+.prj-txt{
```
Comment (cengique) [inline src/styles/global.css:268]: these should be nested inside the .prj-body

File: src/styles/global.css [hunk 7]
```diff
@@ -130,16 +265,74 @@ justify-content: space-around;
...
   display: none;
 }
 .imageLight {
-  display: block;
+  display: flex;
 }
 @media (prefers-color-scheme: dark) {
   .imageDark {
-    display: block;
+    display: flex;
```
Comment (cengique) [inline src/styles/global.css:335]: why flex?

File: src/styles/global.css [hunk 8]
```diff
@@ -71,20 +126,86 @@ main {
...
-h2 {
+h1 {
   text-align:center;
-  font-size: clamp(var(--agnostic-h5), 5vw, var(--agnostic-h2));
+  font-size: clamp(var(--agnostic-h5), 5vw, var(--agnostic-h1));
   font-family: 'Gagalin', Helvetica, Arial, sans-serif;
+  /* text-shadow: black 2px 1px; */
+}
+
+p {
```
Comment (cengique) [inline src/styles/global.css:140]: changing all paragraphs is not a great idea. Please qualify this with where you want this change to occur.

File: src/content/projects/2020-01Spring/DroneReality/DroneReality.mdx [hunk 9]
```diff
@@ -37,8 +37,9 @@ durationMins: [
 publishedDate: 2024-09-13
 relatedIds: []
 image: ""
-imageLogoLight: ./drone.png
-curator: [Trung Dao]
+imageLogoLight: ./drone-reality-logo-light.png
+imageLogoDark: ./drone-reality-logo-dark.png
+curator: [Trung Dao, Kelechi Ariwodo]
 ---
```
Comment (cengique) [inline src/content/projects/2020-01Spring/DroneReality/DroneReality.mdx:44]: this page's layout doesn't look good for some reason.

File: src/components/vue/Breadcrumb.vue (hunk 2 above)
Comment (cengique) [reply src/components/vue/Breadcrumb.vue]: fixed

File: src/styles/global.css (hunk 5 above)
Comment (cengique) [reply src/styles/global.css:87]: removed

File: src/styles/global.css (hunk 7 above)
Comment (cengique) [reply src/styles/global.css:272]: added comment

File: src/styles/global.css (hunk 8 above)
Comment (cengique) [reply src/styles/global.css:141]: @itskelechi this is a major problem. What is this for?

File: src/styles/global.css (hunk 6 above)
Comment (cengique) [reply src/styles/global.css:221]: done

File: src/content/projects/2020-01Spring/DroneReality/DroneReality.mdx (hunk 9 above)
Comment (cengique) [reply src/content/projects/2020-01Spring/DroneReality/DroneReality.mdx:44]: added note

**Review by cengique — APPROVED** [review]
approving my changes

File: src/styles/global.css (hunk 8 above)
Comment (itskelechi) [reply src/styles/global.css:141]: i thought uniform sizing for paragraphs

File: src/components/ProjectCard.vue (hunk 1 above)
Comment (itskelechi) [reply src/components/ProjectCard.vue:162]: I didn't like it at the top but if it's under the title but before the description I can live with it

File: src/styles/global.css (hunk 8 above)
Comment (cengique) [reply src/styles/global.css:141]: > i thought uniform sizing for paragraphs

@itskelechi paragraphs can be used for all kinds of purposes, so again not a good idea to do this site-wide. Please specify which sections of the site this applies. Also, start a new branch from `main` as we merged a lot of stuff together and I modified your branch. Thanks!
